doc/contributing/releases.md

```diff
@@ -605,10 +605,14 @@ the build before moving forward. Use the following list as a baseline:
 * Run the test suite against the built binaries (optional)
 
 ```console
```

Comment:
`console` is for when we want to show both the command and the stdout, here we just have the commands.
````suggestion
```bash
````

Reply:
IMO, I don't think `bash` is a good choice because it will do syntax highlighting which is really weird for a command line. I would just go with `text` or nothing at all. But I don't really have a strong opinion.